source/api-abi-versioning/api-versioning.txt

```diff
@@ -0,0 +1,135 @@
+.. _cpp-api-versioning-policy:
+
+===============
+API Versioning
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+The C++ Driver uses Semantic Versioning (aka SemVer) for API versioning.
+
+Per the `SemVer specification <http://semver.org/>`__, given a version number
+format of MAJOR.MINOR.PATCH, increment vaules when the following changes occur:
+
+- MAJOR
+   when you make incompatible API changes
+- MINOR
+   when you add functionality in a backward compatible manner
+- PATCH
+   when you make backward compatible bug fixes
+
+For purposes of API versioning, we distinguish between the *public API* and the
+*private API*. For this system to work, you first need to declare a public API.
+This may consist of documentation or be enforced by the code itself. Regardless,
+it is important that this API be clear and precise. Once you identify your
+public API, you communicate changes to it with specific increments to your
+version number.
```

Comment:
Let me know what I think of how I did this - we have limited formatting options, and I couldn't get a non-bulleted indent to work.

Reply:
If the documentation generator is using ReStructuredText, it should support [block quotes](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#block-quotes):

```
For purposes of API versioning, we distinguish between the *public API* and the *private API*:

    For this system to work, you first need to declare a public API.
    This may consist of documentation or be enforced by the code itself.
    Regardless, it is important that this API be clear and precise.
    Once you identify your public API, you communicate changes to it with specific increments to your version number.
    
The API version number ...
```